### ISSUE TITLE
Input image size greater than 224

### ISSUE THREAD

**Issue by alejoGT1202** [sipeed/MaixPy3#34]
Hello, 

I could follow successfully the Slim YoloV2 tutorial and deploy a model with my custom data on the V831 board. However, the results where not as expected, so I proceed to increase the input size of my model to 320x320, I was able to convert the model to the format can be deployed on the board but when I tried to run my program on it with the new model, it gave me core dumped. I was wondering if it is actually possible to deploy models with input image size greater than 224x224 or to deploy other models such as TiniYolo V3 on V831 board. Thanks for the help

**Comment by junhuanchen** [comment]
The input depends on the model, not the camera size.

**Comment by junhuanchen** [comment]
目前是否大于 224X224 我没法给你例子，不过我们内部使用是有的，并且我们使用的形状不一定是 224X224。

具体可以请教。@Neutree

**Comment by junhuanchen** [comment]
I can't confirm if it can be bigger than 224X224, but we have other shapes of input.

Specific can consult. @Neutree

**Comment by alejoGT1202** [comment]
Yes, I know the input depends on the model not the camera size, what I did was change the size flag in the `test.py` script, and in fact it gave a `.param` file with input size 320 * 320 then I followed the whole conversion steps to get and optimized `.bin`  and `.param `, Inside my script I had to resize the image captured by the camera to 320x320 but when this new image is passed to the model it is when it crash.

What about using the other models available on this [repo](https://github.com/dianjixz/v831_yolo) I have trained models with YoloV2 architecture and TiniYoloV3 but when I tried to export it using the `test.py` it gives me the following error

```
RuntimeError: Only tuples, lists and Variables are supported as JIT inputs/outputs. Dictionaries and strings are also accepted, but their usage is not recommended. Here, received an input of unsupported type: numpy.ndarray

```

**Comment by ZKH66** [comment]
> 是的，我知道输入取决于模型而不是相机大小，我所做的是更改脚本中的大小标志，实际上它给出了一个输入大小为320 * 320的文件，然后我按照整个转换步骤进行获取和优化，在我的脚本中，我不得不将相机捕获的图像大小调整为320x320，但是当这个新图像传递给模型时，它是当它崩溃时。`test.py``.param``.bin``.param `
> 
> 使用此[存储库](https://github.com/dianjixz/v831_yolo)上可用的其他模型怎么样，我已经使用YoloV2架构和TiniYoloV3训练了模型，但是当我尝试使用它导出它时，它给了我以下错误`test.py`
> 
> ```
> RuntimeError: Only tuples, lists and Variables are supported as JIT inputs/outputs. Dictionaries and strings are also accepted, but their usage is not recommended. Here, received an input of unsupported type: numpy.ndarray
> ```

I would like to ask if this problem is solved

**Comment by ZKH66** [comment]
I would like to ask if this problem is solved

**Comment by alejoGT1202** [comment]
@ZKH66 No I couldn't deploy models with greater input image size

**Comment by soulyiran** [comment]
i also find this problem, seems input feature map have to smaller than or equal to 240x240

**Comment by alejoGT1202** [comment]
Yes, it's to bad because I want to use this board for detecting small objects but with this input resolution the performance it's not good.
